src/index.html

```diff
@@ -17,6 +17,93 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="logo_link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="#apple"
+              class="is-active nav_link"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#samsung"
+              class="nav_link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#smartphones"
+              class="nav_link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#laptops&computers"
+              class="nav_link nav_link-1"
```

Comment:
The class `nav_link-1` does not represent the meaning of the content and seems to be a style-related class name. Consider renaming it to something more meaningful or removing it if it's not necessary.

Reply:
👍